ang/civicase/DashboardTab.html

```diff
@@ -1,5 +1,194 @@
 <div class="civicase__dashboard__tab">
-  <div class="civicase__dashboard__tab__top"></div>
+  <div class="civicase__dashboard__tab__top">
+    <div class="panel panel-secondary civicase__dashboard__overview">
```

Comment:
this section doesn't look like a panel, why are we extending it?

Reply:
Because the section needs styles of `panel-secondary` (box-shadow) and is in sync with other boxes on the page. So if we are using `panel-secondary` we have to add `panel` class.